### DISCUSSION TITLE
failed - Error time out

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/httpx#393] [General]
**Describe the bug**
Error  time out
not correct results 
long time until it finished 
while using  -- >   subfinder -d domain.com | httpx  -o output.txt   

so i tries to use debug mode 

using  -- >subfinder -d domain.com | httpx -debug -o output.txt    
![Capture](https://user-images.githubusercontent.com/85590728/132066585-758be147-0636-48b7-93c0-6b7594874521.PNG)

**Comment by ehsandeep** [comment]
Hi @gang4me,

> Error time out

timeout error is expected for the host not responding after the allowed interval, you can control with `timeout` flag.

> not correct results

Can you please include an example of a valid URL that is missed by httpx?

> long time until it finished

Again it depends on the input list, if you are processing a good number of hosts, it will take time depending upon network and server capacity.
